bgpd/bgp_route.c

```diff
@@ -10551,7 +10559,7 @@ DEFUN (show_bgp_afi_vpn_rd_route,
 	afi_t afi = AFI_MAX;
 	int idx = 0;
 
-	(void)argv_find_and_parse_afi(argv, argc, &idx, &afi);
+	argv_find_and_parse_afi(argv, argc, &idx, &afi);
```

Comment:
Casting the return value of `argv_find_and_parse_afi()` to void was done to silence a Coverity warning. Any reason to undo this?

Reply:
It shouldn't be needed on a function-call though.
It's set correctly on it's definition.

Reply:
When we cast the return value of `argv_find_and_parse_afi()` to void we make it explicit that we're ignoring it. Since we check the return value of this function in other call sites, Coverity Scan assumes we should always do it otherwise we get a "error handling issues" warning.

Reply:
I am not sure why I added this.  Removing.